chsdi/templates/htmlpopup/oev_haltestellen.mako

```diff
@@ -0,0 +1,151 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <%!
+        from pyramid.url import route_url
+        import chsdi.lib.helpers as h
+    %>
+    <%
+        protocol = request.scheme
+        lang = request.lang
+        topic = request.matchdict.get('map')
+        host = h.make_agnostic(request.host_url + request.uscript_name)
+
+        id = c['attributes']['nummer']
+        c['baseUrl'] = h.make_agnostic(''.join((protocol, '://', request.registry.settings['geoadminhost'])))
+    %>
+<style>
+  tr:last-child {
+    display:none;
+  }
+</style>
+<script>
+
+var timesRun = 0;
+var dest;
+function getInfos(val){
+    if (val) {
+      dest = val;
+    }
+    timesRun += 1;
+    if(timesRun === 20){
+      clearInterval(window.refresh);
+    }
+    $.getJSON( "https://mf-chsdi3.dev.bgdi.ch/fap_oev_service/stationboard/stops/${c['attributes']['nummer']}?destination=" + encodeURIComponent(dest), function(result2){
+      var numero = '';
+      var destination = '';
+      var departures = '';
+      var timeDiff = '';
+      for (var i = 0; i < result2.length; i++) {
+        if (dest == 'nodata' || result2[i].destination == 'nodata'){
+          destination += '<p style="height:24px; margin:0px;">${_("ch.bav.haltestellen-oev.nodata2")}</p>';
+        } else {
+          var then = result2[i].time;
+          var now = result2[i].currentDate;
+          var diff = moment.utc(moment(then,"HH:mm").diff(moment(now,"HH:mm"))).format("HH:mm");
+          numero += '<p style="height:24px; margin:0px;">' + result2[i].label + '</p>';
+          destination += '<p style="height:24px; margin:0px;">' + result2[i].destination + '</p>';
+          departures += '<p style="height:24px; margin:0px;">' + result2[i].time + '</p>';
+          timeDiff += '<p style="height:24px; margin:0px;"><b>' + diff +'</b></p>';
+        }
+      };
+      $("#numero").html(numero);
+      $("#destination").html(destination);
+      $("#departures").html(departures);
+      $("#timeDiff").html(timeDiff);
+    }); 
+}
+
+function select(val) {
+  if (window.refresh) {
+    clearInterval(window.refresh);
+  }
+  getInfos(val); 
+  window.refresh = setInterval(getInfos, 60000);
```

Comment:
Also, you can pass in `val` to getInfos everytime here, so you don't need the gloable `dest` variable.